semeio/jobs/overburden_timeshift/ots_config.py

```diff
@@ -101,6 +101,8 @@ def build_schema():
             "velocity_model": {
                 MK.Type: types.String,
                 MK.Description: "Path to segy file containing the velocity field.",
+                MK.Default: None,
```

Comment:
We had several iterations with Lars Petter about whether to allow `None` or not - wrong in the end :)

Reply:
Hehe, the code was always ready for `None`, so I just went with that ;)